aten/src/ATen/core/type.cpp

```diff
@@ -1120,12 +1124,12 @@ std::string ClassType::getForwardPreHookErrorMessage(int pre_hook_idx) const {
   const FunctionSchema& forward_schema = getMethod("forward").getSchema();
   std::string input_types = getSchemaInputTypesString(forward_schema);
   const std::vector<Argument>& forward_args = forward_schema.arguments();
-   
```

Comment:
There seem to be a lot of formatting changes below this point. Can you undo them?

Reply:
they seem inevitable

Reply:
done!